charts/fluent-operator/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
To be honest, it might be better to invert this by having an exclude all and subsequently only include what is required.

Reply:
https://github.com/fluent/helm-charts/blob/main/charts/fluent-bit/.helmignore
https://github.com/fluent/helm-charts/blob/main/charts/fluentd/.helmignore
I have modified it to match the other helm files.